src/ClassDesignChallenge/Professors.java

```diff
@@ -0,0 +1,24 @@
+package ClassDesignChallenge;
+
+import java.time.LocalDate;
+
+public class Professors {
+
+    private String firstName;
+    private String lastName;
+    private char sex;
+    private LocalDate dateOfBirth;
```

Comment:
Do we need sex, dateOfBirth and age properties if we have cnp?

Reply:
Had a thought initially but the other way around, once you have the sex, dateOfBirth and age you can easily have the first 7 digits from the CNP. It also works the other way around. Will take a look again today and amend the code.